RELEASE.md

```diff
@@ -1,39 +1,35 @@
 # Release Process
-zfs-localpv follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/10).
+LocalPV ZFS follows or tries to follow semantic versioning principles as specified here https://semver.org. It follows a on quarterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
 
-## Release Candidate Verification Checklist
+## Pre-release Candidate Verification Checklist
 
-Every release has release candidate builds that are created starting from the third week into the release. These release candidate builds help to freeze the scope and maintain the quality of the release. The release candidate builds will go through:
+Every release has a pre-release version that gets created on branch creation, explained further below. This pre-release version is meant for all the below action items throughout the release process:
```

Comment:
Do we really create a pre-release? I've never seen that..
Or is this about the `pre-release` that gets set as the version on the chart? In that case I think this sentence is a bit misleading.

Reply:
Yes, exactly it's that. Any suggestions please. I am a bit perplexed on how to frame this.